homeassistant/components/ruckus_unleashed/__init__.py

```diff
@@ -24,35 +26,47 @@
 )
 from .coordinator import RuckusUnleashedDataUpdateCoordinator
 
+_LOGGER = logging.getLogger(__package__)
+
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Ruckus Unleashed from a config entry."""
+
     try:
-        ruckus = await Ruckus.create(
+        ruckus = AjaxSession.async_create(
             entry.data[CONF_HOST],
             entry.data[CONF_USERNAME],
             entry.data[CONF_PASSWORD],
         )
-    except ConnectionError as error:
-        raise ConfigEntryNotReady from error
+        await ruckus.login()
+    except ConnectionError as conerr:
+        _LOGGER.exception("ConnectionError: %s", conerr)
+        raise ConfigEntryNotReady from conerr
+    except AuthenticationError as autherr:
+        _LOGGER.exception("AuthenticationError: %s", autherr)
```

Comment:
```suggestion
```

It is already logged, as it is re-raised

Reply:
removed